qcodes/dataset/sqlite_base.py

```diff
@@ -1,2755 +1,47 @@
-
-
-def completed(conn: ConnectionPlus, run_id) -> bool:
-    """ Check if the run scomplete
-
-    Args:
-        conn: database connection
-        run_id: id of the run to check
-    """
-    return bool(select_one_where(conn, "runs", "is_completed",
-                                 "run_id", run_id))
-
-
-def get_completed_timestamp_from_run_id(
-        conn: ConnectionPlus, run_id: int) -> float:
-    """
-    Retrieve the timestamp when the given measurement run was completed
-
-    If the measurement run has not been marked as completed, then the returned
-    value is None.
-
-    Args:
-        conn: database connection
-        run_id: id of the run
-
-    Returns:
-        timestamp in seconds since the Epoch, or None
-    """
-    return select_one_where(conn, "runs", "completed_timestamp",
-                            "run_id", run_id)
-
-
-def get_guid_from_run_id(conn: ConnectionPlus, run_id: int) -> str:
-    """
-    Get the guid of the given run
-
-    Args:
-        conn: database connection
-        run_id: id of the run
-    """
-    return select_one_where(conn, "runs", "guid", "run_id", run_id)
-
-
-def finish_experiment(conn: ConnectionPlus, exp_id: int):
-    """ Finish experiment
-
-    Args:
-        conn: database connection
-        name: the name of the experiment
-    """
-    query = """
-    UPDATE experiments SET end_time=? WHERE exp_id=?;
-    """
-    atomic_transaction(conn, query, time.time(), exp_id)
-
-
-def get_run_counter(conn: ConnectionPlus, exp_id: int) -> int:
-    """ Get the experiment run counter
-
-    Args:
-        conn: the connection to the sqlite database
-        exp_id: experiment identifier
-
-    Returns:
-        the exepriment run counter
-
-    """
-    return select_one_where(conn, "experiments", "run_counter",
-                            where_column="exp_id",
-                            where_value=exp_id)
-
-
-def get_experiments(conn: ConnectionPlus) -> List[sqlite3.Row]:
-    """ Get a list of experiments
-     Args:
-         conn: database connection
-
-     Returns:
-         list of rows
-     """
-    sql = """
-    SELECT * FROM experiments
-    """
-    c = atomic_transaction(conn, sql)
-
-    return c.fetchall()
-
-
-def get_matching_exp_ids(conn: ConnectionPlus, **match_conditions) -> List:
-    """
-    Get exp_ids for experiments matching the match_conditions
-
-    Raises:
-        ValueError if a match_condition that is not "name", "sample_name",
-        "format_string", "run_counter", "start_time", or "end_time"
-    """
-    valid_conditions = ["name", "sample_name", "start_time", "end_time",
-                        "run_counter", "format_string"]
-
-    for mcond in match_conditions:
-        if mcond not in valid_conditions:
-            raise ValueError(f"{mcond} is not a valid match condition.")
-
-    end_time = match_conditions.get('end_time', None)
-    time_eq = "=" if end_time is not None else "IS"
-
-    sample_name = match_conditi
```

Comment:
As above should this raise a warning on import?

Reply:
yes, good point. i'll add a warning